script.js

```diff
@@ -1,41 +1,79 @@
 const playAgainButton = document.querySelector("#playagain");
-const h1 = document.querySelector("#heading");
+const head = document.querySelector("#heading");
 const yourScore = document.querySelector("#score");
 const blankStart = document.querySelector("#blank");
 const diceScore = document.querySelector("#dice");
 const rollButton = document.querySelector("#submit");
 
-
-let randomNumber = Math.floor(Math.random()*(7-1)+1);
 let totScore = 0;
 
-
+// if (randomNumber == 1){
+//     head.textContent= "You Lost"
+//     totScore = 0;
+//     playAgainButton.textContent = "Play Again";
+// }
 
 
 rollButton.addEventListener("click", () => {
+    let randomNumber = Math.ceil(Math.random()* 6 );
```

Comment:
Will you occasionally get 0s rolled here? Or does the `ceil` avoid that?

Reply:
Ceil rounds up, so 0's are not possible. Math.random()* 6 is creating numbers in the range 0-6 (but not including 0 or 6 ); the ceil is then making those 1 through 6.